src/ansys/aedt/toolkits/antenna/backend/run_backend.py

```diff
@@ -0,0 +1,137 @@
+# Copyright (C) 2023 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+import sys
+
+# isort: off
+
+from ansys.aedt.toolkits.antenna.backend.api import ToolkitBackend
+
+from ansys.aedt.toolkits.common.backend.multithreading_server import MultithreadingServer
+from ansys.aedt.toolkits.common.backend.rest_api import app
+from ansys.aedt.toolkits.common.backend.rest_api import jsonify
+from ansys.aedt.toolkits.common.backend.rest_api import logger
+
+from flask import request
+
+# isort: on
+
+toolkit_api = ToolkitBackend()
+
+if len(sys.argv) == 3:
+    toolkit_api.properties.url = sys.argv[1]
+    toolkit_api.properties.port = int(sys.argv[2])
+
+
+@app.route("/create_antenna", methods=["POST"])
+def create_antenna():
+    logger.info("[POST] /create_antenna (create antenna in HFSS)")
+
+    properties = toolkit_api.get_properties()
+
+    antenna = properties["antenna"]["model"]
+    synth_only = properties["antenna"]["synth_only"]
+
+    response = toolkit_api.get_antenna(antenna, synth_only=synth_only)
+    if response:
+        return jsonify(response), 200
+    else:  # pragma: no cover
+        return jsonify("Antenna not created"), 500
+
+
+@app.route("/update_parameters", methods=["PUT"])
+def update_parameters():
+    logger.info("[POST] /update_parameters (Update parameters in HFSS)")
```

Comment:
I would recommend renaming the end point to "hfss_parameters" (might not be only used for hfss but that's my feeling when reading the file) to have better resource name.
Also I would change the method from PUT to PATCH since the code is update a subset (here only one) of all existing parameters. Using PUT as HTTP verb should only be performed when updating all parameters at once.

Reply:
Btw, the logger should be updated with the right verb too